src/app/loadout-drawer/loadout-events.ts

```diff
@@ -32,3 +37,17 @@ export function editLoadout(
 export function addItemToLoadout(item: DimItem) {
   addItem$.next(item);
 }
+
+/**
+ * Copy and Edit Loadout
+ */
+export function copyAndEditLoadout(
+  loadout: Loadout,
+  storeId: string,
+  { showClass = true }: { showClass?: boolean } = {},
+) {
+  const copiedLoadout = structuredClone(loadout);
+  copiedLoadout.name = `${loadout.name} - Copy`;
```

Comment:
This should probably be
```suggestion
  const copiedLoadout = { ...loadout, name: `${loadout.name} - Copy` };
```
since our Loadouts are treated as immutable anyway?